<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formSquareApertureSettings</class>
 <widget class="QDialog" name="formSquareApertureSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fluorescent Wavelengths</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Wavelen Blue</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxBlue">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>16</y>
     <width>70</width>
     <height>29</height>
    </rect>
   </property>
   <property name="minimum">
    <double>300.000000000000000</double>
   </property>
   <property name="maximum">
    <double>510.000000000000000</double>
   </property>
   <property name="value">
    <double>438.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>55</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Wavelen Green</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxGreen">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>51</y>
     <width>70</width>
     <height>29</height>
    </rect>
   </property>
   <property name="minimum">
    <double>450.000000000000000</double>
   </property>
   <property name="maximum">
    <double>570.000000000000000</double>
   </property>
   <property name="value">
    <double>542.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Wavelen Red</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxRed">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>86</y>
     <width>70</width>
     <height>29</height>
    </rect>
   </property>
   <property name="minimum">
    <double>500.000000000000000</double>
   </property>
   <property name="maximum">
    <double>750.000000000000000</double>
   </property>
   <property name="value">
    <double>618.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>124</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>EM Iterations</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxEMIterations">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>121</y>
     <width>71</width>
     <height>29</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>formSquareApertureSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>formSquareApertureSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
